### PR TITLE
Feat/netapp volumes

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/netapp-dashboard`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7147
| License          | BSD 3-Clause

## Description
Add volumes list page

### CODE REVIEW

File: packages/manager/modules/netapp/src/dashboard/volumes/routing.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        $q.all(
+          volumes.map((volume) =>
+            $q
+              .all({
+                shareSnapshotPolicy: $http
+                  .get(
+                    `/storage/netapp/${serviceName}/share/${volume.id}/snapshotPolicy`,
+                  )
+                  .then(({ data }) => data)
+                  .catch(() => null),
```
Comment (JeremyDec) [inline packages/manager/modules/netapp/src/dashboard/volumes/routing.js:20]: Waiting API route fix to know if this is required

File: packages/manager/modules/netapp/src/dashboard/volumes/routing.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import Share from './Share.class';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.dashboard.volumes', {
+    url: '/volumes',
+    views: {
+      '@netapp.dashboard': {
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/dashboard/volumes/routing.js:7]: Isn't this the default behaviour ? 
```suggestion
```

File: packages/manager/modules/netapp/src/dashboard/volumes/routing.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        component: 'ovhManagerNetAppVolumes',
+      },
+    },
+    resolve: {
+      detailedVolumes: /* @ngInject */ ($http, $q, serviceName, volumes) =>
+        $q.all(
+          volumes.map((volume) =>
+            $q
+              .all({
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/dashboard/volumes/routing.js:16]: Wouldn't it be better to use a `row-loader` instead ?

Comment (JeremyDec) [reply packages/manager/modules/netapp/src/dashboard/volumes/routing.js]: Done, thanks

File: packages/manager/modules/netapp/src/dashboard/volumes/routing.js (hunk 2 above)
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/dashboard/volumes/routing.js]: Yep you're right, old habit to define absolute path for views

**Review by frenautvh — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Comment by marie-j** [comment]
Rebase on feat/netapp

**Pull request merged**